addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -48,6 +49,7 @@ export class PropertiesField extends Component {
         this.state = useState({
             canChangeDefinition: true,
             movedPropertyName: null,
+            hideAddButton: this.props.hideAddButton,
```

Comment:
I think you either need to specify a default value or do "!!" here to turn this into a boolean?
(Better to specify the default if possible).

Reply:
It does indeed, so not necessary to specify.